introduction/architecture.md

```diff
@@ -111,6 +110,64 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
 
+## TimescaleDB Clustering [](timescaledb-clustering) (PRIVATE BETA)
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
+In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
+depending on their responsibilities within a distributed database.
+
+The two roles that nodes can take on are:
+1) Access node
+2) Data node
+
+A front-end access node accepts all requests from the client, distributes the requests and queries
```

Comment:
I would say to even add an h3 for "Node types" or an h4 headline for each of the node types

Reply:
I added in an h3 for Node Types